### PR TITLE
Fixing multi host

### PR DESCRIPTION
**Pull request by SeanFeldman**
Fixes issue #30 

@Particular/azure-maintainers please review

@yvesgoeleven there seems to be an assumption about NServiceBus.Hosting.Azure.HostProcess.exe not being scanned by `AssemblyScanner`, but it does get scanned...

### CODE REVIEW

File: src/NServiceBus.Hosting.Azure.HostProcess/NServiceBus.Hosting.Azure.HostProcess.csproj [hunk 1]
```diff
@@ -123,7 +125,7 @@
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Target Name="ReleaseAfterBuild" AfterTargets="AfterBuild" Condition=" '$(Configuration)' == 'Release' ">
     <MakeDir Directories="$(IntermediateOutputPath)temp\" />
-    <Exec Command="&quot;$(SolutionDir)..\tools\IlMerge\ilmerge.exe&quot; /keyfile:&quot;$(SolutionDir)NServiceBus.snk&quot; /internalize /out:&quot;$(IntermediateOutputPath)temp\$(TargetFileName)&quot; &quot;$(TargetPath)&quot; &quot;$(TargetDir)log4net.dll&quot; &quot;$(TargetDir)Microsoft.Practices.ServiceLocation.dll&quot; &quot;$(TargetDir)Topshelf.dll&quot; /targetplatform:&quot;v4,$(FrameworkPathOverride)&quot;" />
+    <Exec Command="&quot;$(SolutionDir)packages\ILMerge.2.14.1208\tools\ilmerge.exe&quot; /keyfile:&quot;$(SolutionDir)NServiceBus.snk&quot; /internalize /out:&quot;$(IntermediateOutputPath)temp\$(TargetFileName)&quot; &quot;$(TargetPath)&quot; &quot;$(TargetDir)log4net.dll&quot; &quot;$(TargetDir)Microsoft.Practices.ServiceLocation.dll&quot; &quot;$(TargetDir)Topshelf.dll&quot; /targetplatform:&quot;v4,$(FrameworkPathOverride)&quot;" />
```
Comment (SimonCropp) [inline src/NServiceBus.Hosting.Azure.HostProcess/NServiceBus.Hosting.Azure.HostProcess.csproj]: this is fragile since it will break with a nuget update

Comment (SeanFeldman) [inline src/NServiceBus.Hosting.Azure.HostProcess/NServiceBus.Hosting.Azure.HostProcess.csproj]: This was intentional. Spent a good chunk of time chasing what's causing a "No RVA" exception.

Comment (SeanFeldman) [inline src/NServiceBus.Hosting.Azure.HostProcess/NServiceBus.Hosting.Azure.HostProcess.csproj]: Nuget updates once a year or two :)

Comment (SimonCropp) [inline src/NServiceBus.Hosting.Azure.HostProcess/NServiceBus.Hosting.Azure.HostProcess.csproj]: fair enough. can u send a PR to the core with the same change

**Comment by SimonCropp** [comment]
this looks all good. assume u have smoke tested?

**Comment by SeanFeldman** [comment]
Yep.

File: src/NServiceBus.Hosting.Azure.HostProcess/NServiceBus.Hosting.Azure.HostProcess.csproj (hunk 1 above)
Comment (SeanFeldman) [inline src/NServiceBus.Hosting.Azure.HostProcess/NServiceBus.Hosting.Azure.HostProcess.csproj]: Will do

**Comment by SimonCropp** [comment]
merging since it was fundamentally broken prior to this

**Pull request merged**